src/pages/teamMembers.js

```diff
@@ -0,0 +1,245 @@
+    emailid: "abcdef@gmail.com"
+  },
+  {
+    name: "12",
+    designation: "Co-Founder",
+    img: "https://source.unsplash.com/random/200x200",
+    githublink: "github.com/pqrst",
+    linkedin: "linkedin.com/in/pqrst",
+    phonenNo: "9898989898",
+    emailid: "pqrst@gmail.com"
+  },
+  {
+    name: "13",
+    designation: "President",
+    img: "https://source.unsplash.com/random/200x200",
+    githublink: "github.com/abcdef",
+    linkedin: "linkedin.com/in/abcdef",
+    phonenNo: "9898989898",
+    emailid: "abcdef@gmail.com"
+  },
+  {
+    name: "14",
+    designation: "Member",
+    img: "https://source.unsplash.com/random/200x200",
+    githublink: "github.com/pqrst",
+    linkedin: "linkedin.com/in/pqrst",
+    phonenNo: "9898989898",
+    emailid: "pqrst@gmail.com"
+  }
+]
+
+export default function teamMembers() {
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(false);
+  const [index, setIndex] = React.useState(0);
+
+  const handleOpen = (data) => {
+    setOpen(true);
+    if (data != undefined) {
+      setIndex(data);
+    }
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <Fragment>
+      <div className={classes.heading}>
+        <Container maxWidth="md">
+          <div>
+            <h1><u>OUR TEAM</u></h1>
+          </div>
+          <div className={classes.root} >
+            <GridList cellHeight={300} spacing={40} cols={3} className={classes.gridTile}>
+              {team.map((tile, i) => (
+                <GridListTile key={team.indexOf(tile)} >
+                  <img src={tile.img} alt={tile.name} className={classes.imageFullWidth} />
+                  <GridListTileBar
+                    title={tile.name}
+                    subtitle={<span>{tile.designation}</span>}
+                    actionIcon={
+                      <Fragment>
+                        <button type="button" onClick={() => handleOpen(i)}>i</button>
+                        <Modal
+                          // aria-labelledby="transition-modal-title"
+                          // aria-describedby="transition-modal-description"
+                          className={classes.modal}
+                          open={open}
+                          onClose={handleClose}
+                          closeAfterTransition
+                          BackdropComponent={Backdrop}
+                          BackdropProps={{
+                            timeout: 500,
+                            style: {
+                              backgroundColor: "transparent"
+                            }
+                          }}
+                        >
+                          <Zoom in={open}>
+                            <div className={classes.paper}>
+                              <h1 id="transition-modal-title" className={classes.heading}>CONTACT</h1>
+                              <h2 id="transition-modal-title"><span>Phone: {team[index].phonenNo}</span></h2>
```

Comment:
We can use 'tile' variable instead of team[index] as we have declared it in Map function.